app/__init__.py

```diff
@@ -1,7 +1,31 @@
 from flask import Flask
+# Step 1:
+# Import and initialize SQLAlchemy
+from flask_sqlalchemy import SQLAlchemy
+from flask_migrate import Migrate
 
+# Initialize SQLAlchemy
+db = SQLAlchemy()
+migrate = Migrate()
+DATABASE_CONNECTION_STRING='postgresql+psycopg2://postgres:postgres@localhost:5432/solar_system_development'
```

Comment:
I suggest in the future storing the connection strings in environment variables.